.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,30 @@
+name: CIFuzz
```

Comment:
Can we please rename the workflow file to `tests-fuzz.yaml` for consistency with the other workflows filename convention?

Reply:
Done!